src/schemas/intents.json

```diff
@@ -342,6 +345,37 @@
         }
       ]
     },
+    "PreLoadIntent": {
+      "description": "A Firebolt compliant representation of a user intention to pre-load an app.",
```

Comment:
We can probably remove the 'user intention' as a user wouldn't be responsible for pre-loading an app.

Reply:
agree